test/helpers/cmd.go

```diff
@@ -226,3 +244,31 @@ func (res *CmdRes) WaitUntilMatch(substr string) error {
 		fmt.Sprintf("%s is not in the output after timeout", substr),
 		&TimeoutConfig{Timeout: HelperTimeout})
 }
+
+// BeSuccesfulMatcher a new Ginkgo matcher for CmdRes struct
+type BeSuccesfulMatcher struct{}
+
+func (matcher *BeSuccesfulMatcher) Match(actual interface{}) (success bool, err error) {
+	res, ok := actual.(*CmdRes)
+	if !ok {
+		return false, fmt.Errorf("%q is not a valid *CmdRes type", actual)
+	}
+	return res.WasSuccessful(), nil
+}
+
+func (matcher *BeSuccesfulMatcher) FailureMessage(actual interface{}) (message string) {
+	res, _ := actual.(*CmdRes)
+	fmt.Println(res.GetCmd())
+	return fmt.Sprintf("Expected command: %s \nTo succeed, but it fails:\n%s",
+		res.GetCmd(), res.OutputPrettyPrint())
+}
+
+func (matcher *BeSuccesfulMatcher) NegatedFailureMessage(actual interface{}) (message string) {
```

Comment:
NegatedFailureMessage --> NegativeFailureMessage ?

Reply:
I can't change this one, required by Ginkgo/Gomega ;-)